<ui version="4.0" >
 <class>MediaflyPersonalize</class>
 <widget class="QWidget" name="MediaflyPersonalize" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QStackedWidget" name="m_stackedWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_0" >
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Personalize</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer name="verticalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>399</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>2</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Personalize the channels on your Neuros by linking to your FREE Mediafly.com account.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>- Select your favorite channels or create your own</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
         <property name="indent" >
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>- Find shows you love from the networks you love</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
         <property name="indent" >
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="9" column="0" >
        <widget class="QLabel" name="label_6" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>- Discover popular new shows from independent creators</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
         <property name="indent" >
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0" >
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>399</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="11" column="0" >
        <widget class="QPushButton" name="m_ContinueButton" >
         <property name="font" >
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Continue setup</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="font" >
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Add a new Mediafly profile link</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <spacer name="verticalSpacer_5" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0" >
        <spacer name="verticalSpacer_6" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_1" >
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_7" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Personalize</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_8" >
         <property name="font" >
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>To create a Mediafly account, follow these simple steps.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_9" >
         <property name="font" >
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Step 1: Visit www.Mediafly.com from any computer.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
         <property name="indent" >
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_10" >
         <property name="font" >
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Step 2: Create a free account.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
         <property name="indent" >
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QLabel" name="label_11" >
         <property name="font" >
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Step 3: Enter your Username and Password on the next screen.</string>
         </property>
         <property name="wordWrap" >
          <bool>true</bool>
         </property>
         <property name="indent" >
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0" >
        <widget class="QPushButton" name="m_EnterButton" >
         <property name="font" >
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Enter Username and Password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
